lab_1/main.py

```diff
@@ -36,7 +63,14 @@ def calculate_frequencies(tokens: list) -> dict:
     e.g. tokens = ['weather', 'sunny', 'man', 'happy']
     --> {'weather': 1, 'sunny': 1, 'man': 1, 'happy': 1}
     """
-    pass
+    dictionary = {}
+    freq = 0
+    for b in tokens:
+        freq = tokens.count(b)
+        dictionary[b] = freq
+        result = dictionary
```

Comment:
You do this every time the `for`-loop is executed. I think, it should only be done once - after `for`-loop.